syft/workers/base.py

```diff
@@ -182,7 +182,7 @@ def __init__(
         # storage object for crypto primitives
         self.crypto_store = PrimitiveStorage(owner=self)
         # declare the plans used for crypto computations
-        sy.frameworks.torch.mpc.fss.initialize_crypto_plans(self)
+        # sy.frameworks.torch.mpc.fss.initialize_crypto_plans(self)
```

Comment:
Should be uncommented?

Reply:
That actually needs to get fixed. There's a separate issue for it: #3464

Reply:
Oops, I had commented that only for local debug, thanks for catching that!